Adds Crime Scene Analysts to Security

FearTheBlackout opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->
- Crime Scene Analysts are the forensic contractor equivalent of Forensic Technicians. They are a separate role from the Forensic Technician and are NOT meant to provide as law enforcement.
- They receive their own locker in Forensics with a flash and pepperspray, but no taser or stun baton to emphasize this. They cannot access the Brig.

hey-it-stev commented:
Don't see the point of splitting it into two roles - why not give the ftech slot an extra slot and make it open to both?

I mean I wouldn't mind this implementation too much and I want this to happen, just seems like a slightly worse way of doing it than the alternative.

ghost commented:
Yeah but the point was to have more contractors in Security. This prevents there from being two FTs at the same time.
I'm open to doing it either way, really.

hey-it-stev commented:
Seems to me it'd just be a bit simpler to merge the two. For one thing it'd help reduce slot competition, which is always nice.

ghost commented:
I'll alter it for there simply to be 2 Forensic Technician slots if asked to after review.

AtomicWorm commented:
So why would a contractor specifically contracted to do investigations get the same kit as a military LEO?

ghost commented:
People don't like it when I try to give them less equipment or I would probably get rid of the taser.

AtomicWorm commented:
Yeah? Well other people also don't like it when you try to shove in contractors into security.

ghost commented:
Not really my problem. People had an idea and I decided to implement it.

hey-it-stev commented:
Other people also don't like being arbitrarily restricted from a good portion of the game's content because of a differing playstyle.

Minijar commented:
I'm in favour of FT being a contractor/mil role but this is just poor execution. Splitting the role is awful as there is NEVER enough work for one most times, let alone two people, this is what happened on Exodus all the time, the other always stole the others work all the time leaving one with nothing to do.

comma commented:
Just add contractor branch to existing job, this is a silly way to go about it.

ghost commented:
One slot or two? I've heard arguments in favor of both.

comma commented:
One

ghost commented:
Okay this should be good to review.

PurpleMartinJCK commented:
This needs a squash. If you need help, please ask rather than making a new PR.

ghost commented:
Yeah that doesn't seem to work properly with Gitkraken and I don't feel like trying something else.

PurpleMartinJCK commented:
https://blog.axosoft.com/learning-git-gitkraken-squash-commits/

Rebruiza commented:
![image](https://user-images.githubusercontent.com/31956636/34416055-f75bd7ea-ebf1-11e7-92f1-56469f9339ab.png)
:thinking:

ghost commented:
Yeah that just makes more commits.

AtomicWorm commented:
How convenient that is for you.

Techhead0 commented:
Some non-constructive comments have been removed. Carry on.

ghost commented:
**I DID IT**

AtomicWorm commented:
I'm gonna be putting up an alternative to this soon, so, if merging can be held off, that'd be great.

BlueNexus commented:
https://github.com/Baystation12/Baystation12/pull/19969 Looks better, imo.

ZaCyril commented:
Just wanted to point at this thread with poll ( https://baystation12.net/forums/threads/open-ft-slot-for-contractors.5927/ ) is about Contractor and only, which Colonel Marshals aren't whatsoever - #19969 isn't an alternative, it's another role.

Zero-Bits commented:
I personally like #19969 more and hope it gets implemented after this is handled. That said, I don't have anything against this PR. It seems alright.

PurpleMartinJCK commented:
This one merging doesn't exclude the other one from also being merged. This one does need to go through first though.

Nerev4r commented:
Yeah, I also like #19969 more.

PurpleMartinJCK commented:
I've spoken with a few people and will be handling these PR's a different way. Stand by.

AtomicWorm commented:
I like this now.

PurpleMartinJCK commented:
Still got some conflicts

ZaCyril commented:
I'd suggest providing them basic medical access in order for them to be able to use DNA search function on consoles as well as get to the mourge not only through maint tunnels but medical hall too.

babydoll commented:
Really don't see the point in adding another FT slot

AtomicWorm commented:
Not the same as the current FT. And the point is people wanted contractors in security, this is the way for them to be added that makes sense.

babydoll commented:
Can change the name all you want but it doesn't change that they're just detectives with a different name, in a different slot. Chemist and virologist have a better claim to being a unique slot than this.

Techhead0 commented:
>Can change the name all you want but it doesn't change that they're just detectives with a different name, in a different slot. Chemist and virologist have a better claim to being a unique slot than this.

Yeah, as a Dev this is my main beef with the role. Secondarily, the Torch barely has enough crime scenes for one person, let alone two.

PurpleMartinJCK commented:
@Techhead0 it's because FT has the ability to do law enforcement stuff and this one isn't supposed to do so.

babydoll commented:
What's stopping them?

PurpleMartinJCK commented:
They won't have the access or ability to do so.

babydoll commented:
Are you going to make it so you need access to use handcuffs or a taser? If not, how are you going to prevent them from arresting people, or just being another security officer slot 'when needed'?

PurpleMartinJCK commented:
Yeah they're not going to have access to them and will be no more a security slot when needed as any other contractor

babydoll commented:
Is the admin team prepared to aggressively ensure that people play within the rules established here, then?

AtomicWorm commented:
They don't even GET a taser. Look at what their locker will contain.

babydoll commented:
If you put someone in the security department with a security radio they are going to be given a weapon and they are going to devolve into a 9th(?) security officer slot. This happened with detective, FT, and it's going to happen with this.

AtomicWorm commented:
Maybe. Maybe not. We'll see. Nothing stops it from being removed if it becomes a problem.

hey-it-stev commented:
If an extra slot is such an issue just make 'Contractor' a selectable branch.

FearTheBlackout commented:
Okay. This is FINALLY ready for review.

Commit:
Adds Crime Scene Analysts

Adds Crime Scene Analysts


Revert "SolGov Rep Badge"

This reverts commit 69886f15909351bb32c46f403f4a4930172d3883.

Merge pull request #19946 from sabiram/revert-19914-merrychristmas

Revert "Add a Christmas tree to the galley"
Merge branch 'dev' into feartheblackout/rideemcowboy
Merge remote-tracking branch 'origin/feartheblackout/rideemcowboy' into feartheblackout/rideemcowboy


Revert "Adds Crime Scene Analysts"

This reverts commit d032431adf5ec85afcec7b36492befdb13adeb6a.

EcklesFire commented:
Honestly I don't think having EC/Fleet/Marines AND Cmars and Contractors fighting for the same, lone slot is a good thing.

Nerev4r commented:
This is another slot @EcklesFire

ghost commented:
^

babydoll commented:
There's absolutely no such thing as 'fighting for a slot'. You ready up and the game assigns players to slots based on their preferences. Every other job in the game has this system.

ghost commented:
OP edited to provide more clarity.

EcklesFire commented:
If it has a separate slot then all good.

@sabiram When I say "fighting for a slot" I mean "you have one, lone slot,  and multiple people fight for that specific role.

With things like captain, fair do's because it's the kind of thing you most certainly should have one of.

With something as low-energy/casual as Ftech, not so much.

ghost commented:
Okay now this is good to review for real this time.

babydoll commented:
@EcklesFire But this isn't any different than any other slot. We have slots limited in number for a good reason, and we should pay extra attention to the number of slots in security in particular. 

There's very, very little real work for detectives to be doing in the first place, so why add a second one? We're already occasionally seeing PIs (THE ASSISTANT SLOT) being used as back up security.

ThatOneGuy42 commented:
I'm against this. So very against this.

ghost commented:
why not just make the FT a contractor job? i don't see a reason why is was military anyway

anyway, i support this

comma commented on maps/torch/job/outfits.dm:
```diff
@@ -167,6 +167,7 @@ Keeping them simple for now, just spawning with basic EC uniforms, and pretty mu
 	shoes = /obj/item/clothing/shoes/laceup
 	id_type = /obj/item/weapon/card/id/torch/crew/representative
 	pda_type = /obj/item/device/pda/heads
+	backpack_contents = list(/obj/item/clothing/accessory/badge/solgov/representative = 1)
```
???

FearTheBlackout replied:
oh my god why is that still in there

ThatOneGuy42 commented:
Ah I didn't realize that this was getting merged, if I would have, I would have veto'd this before the PR came out. Whoops. 

Still tho would like to veto cause apparently I have to say the word veto.

Yes I can do this.

ghost commented:
why tho

Minijar commented:
Pls revert, this was implemented terribly. Add this as an option for orensic Tech, so they have the option of being civillian or military or Colonial Marshal but in the same slot like the chef can be.

ghost commented:
6 upvotes/11 downvotes. Still merged. :thinking:

ghost commented:
You mean reactions are meaningless and Devs get the final say? Who knew?! 😲

ThatOneGuy42 commented:
Because this actually one of the worst ideas I’ve ever seen proposed for a security role in how utterly useless the CSA is aboard the ship and the 0 reason for it. 

(Also I can veto gameplay changes that I think fuck yo the game and setting)
